.github/workflows/main_lastman-standing-app.yml

```diff
@@ -16,6 +16,9 @@ jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
 
+    env:
+        LMS_ENV_INFORMATION: ${{ secrets.LMS_ENV_INFORMATION}}
+

```

Comment:
Just need to get rid of this from the Pull request and it should be fine, good work caoimhe